src/main/java/uk/nhs/digital/nhsconnect/nhais/model/jsonpatch/AmendmentPatch.java

```diff
@@ -8,6 +8,8 @@
 import lombok.Setter;
 import lombok.ToString;
 
+import org.apache.logging.log4j.util.Strings;
```

Comment:
I suspect this is not the Strings class you want

Reply:
no, I will just add a constant instead :)